### PR TITLE
Improvements/fixes

### PR DESCRIPTION
**Pull request by armujahid**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "useTabs": false,
     "singleQuote": true,
-    "tabWidth": 4,
```
Comment (armujahid) [inline .prettierrc]: This can be reverted.

File: apps/api/src/main.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { greet } from '@ts-setup/core';
+import { greet } from '@ts-monorepo/core';
```
Comment (armujahid) [inline apps/api/src/main.ts:1]: core import has been fixed in all files

File: package.json [hunk 3]
```diff
@@ -16,7 +16,8 @@
     "scripts": {
         "preinstall": "npx only-allow pnpm",
         "lint": "eslint .",
-        "test": "pnpm jest --passWithNoTests"
+        "test": "pnpm jest --passWithNoTests",
+        "format": "prettier --write \"packages/**/*.ts\" \"apps/**/*.ts\""
```
Comment (armujahid) [inline package.json:20]: format command has been added

File: tsconfig.json [hunk 4]
```diff
@@ -7,10 +7,7 @@
         "forceConsistentCasingInFileNames": true,
         "skipLibCheck": true,
         "sourceMap": true,
-        "moduleResolution": "node",
-        "paths": {
-            "@ts-setup/core": ["packages/core"]
```
Comment (armujahid) [inline tsconfig.json:12]: This paths is not required. I am not sure why this was present. I have removed this and tested all 3 apps

Comment (vomaksh) [reply tsconfig.json:12]: ``paths`` attribute helps when you are importing in a monorepo. In my case, when I was importing stuff in vscode then a couple of times, these packages were imported as relative path files. By setting this attribute, that relative path imports problem was solved.

I don't know if I was able to explain it. I will share a link regarding it.
https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json

File: package.json (hunk 3 above)
Comment (vomaksh) [reply package.json:20]: I have added prettier formatter in eslint configuration.
So, I think we can use ``eslint --fix .`` but I need to check if it works as expected.

File: apps/api/src/main.ts (hunk 2 above)
Comment (vomaksh) [reply apps/api/src/main.ts:1]: Thanks for fixing embarrassing errors 🙏

File: .prettierrc (hunk 1 above)
Comment (vomaksh) [reply .prettierrc]: Yes please revert this change

**Comment by vomaksh** [comment]
Please add your name in ``README.md`` in Authors section

**Pull request merged**